Xamarin.Forms.Core/XamlParseException.cs

```diff
@@ -3,11 +3,36 @@
 
 namespace Xamarin.Forms.Xaml
 {
+#if NETSTANDARD2_0
+	[Serializable]
+#endif
 	public class XamlParseException : Exception
 	{
 		readonly string _unformattedMessage;
 
-		public XamlParseException(string message, IXmlLineInfo xmlInfo, Exception innerException = null) : base(FormatMessage(message, xmlInfo), innerException)
+		public XamlParseException()
+		{
+		}
+
+		public XamlParseException(string message)
+			: base(message)
+		{
+		}
+
+		public XamlParseException(string message, Exception innerException)
+			: base(message, innerException)
+		{
+		}
+
+#if NETSTANDARD2_0
+		protected XamlParseException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context)
+			: base(info, context)
+		{
+		}
+#endif
+
+		public XamlParseException(string message, IXmlLineInfo xmlInfo, Exception innerException = null)
+			: this(FormatMessage(message, xmlInfo), innerException)
```

Comment:
should be `base` instead of `this`

Reply:
That was calling **_base_** in fact. But I changed it to **_this_**.

The result would be the same, because this(_message_,_innerException_) will call base(_message_,_innerException_) eventualy.

I'm open to change that to **_base_** anyway.

Is there anything else I can do to improve the code?

Reply:
> I'm open to change that to base anyway.

please do

Reply:
@StephaneDelcroix Done